index.html

```diff
@@ -33,5 +42,139 @@ <h3>I'm a software Developer, </h3>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <ul class="profesional-holder">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">Bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+      </div>
+      <button class="project-btn"><a href="#">See project</a></button>
+    </section>
+    <section class="section-three">
+      <div class="background-holder">
+        <h1>Profesional Art Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <ul class="profesional-holder">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">Bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+      </div>
+      <button class="project-btn"><a href="#">See project</a></button>
+    </section>
+    <section class="section-three">
+      <div class="background-holder">
+        <h1>Profesional Art Printing Data</h1>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <ul class="profesional-holder">
+          <li><a href="#">HTML</a></li>
+          <li><a href="#">Bootstrap</a></li>
+          <li><a href="#">Ruby</a></li>
+        </ul>
+      </div>
+      <button class="project-btn"><a href="#">See project</a></button>
+    </section>
+    <section class="section-four">
+      <div class="about-me-ctn">
+        <h1>About me</h1>
+        <p>
+          Hello I&apos;m a software developer! I can help you build a product , feature or website Look through some of my work and experience!
+          If you like what you see and have a project you need coded, don&apos;t hesitate to contact me.
+        </p>
+        <button><a href="#">Get my resume</a></button>
+      </div>
+    </section>
+    <section class="section-five">
+      <img class="divider" src="./final-images/Devider.png" alt="">
+      <div class="language-container">
+         <h2><img src="./final-images/Rectangle 55.png" alt="">Language</h2>
+         <ul>
+          <li>JavaScript</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+         </ul>
+      </div>
+       <div class="frameworks-container">
+          <h2><img src="./final-images/Rectangle 56.png" alt="">Frameworks</h2>
+          <ul>
+           <li>React.js</li>
+           <li>Ruby on Rails</li>
+           <li>RSpec</li>
+           <li>Capybara</li>
+          </ul>
+       </div>
+       <div class="skills-container">
+          <h2><img src="./final-images/Rectangle 57.png" alt="">Skills</h2>
+          <ul>
+           <li>Database Management</li>
+           <li>Version Control</li>
+           <li>CLI</li>
+           <li>API Design</li>
+           <li>Web Development</li>
+          </ul>
+       </div>
+    </section>
```

Comment:
- [ ] Good observation! It's important to ensure that your design is responsive and adapts well to different screen sizes. Instead of using fixed widths and heights for your elements, I recommend using relative units such as percentages or viewport units (vw, vh) to allow for fluid responsiveness. For the mobile version, aim to make your design look good on a minimum screen width of 375px. Test and adjust your layout to ensure it remains responsive up to a desktop screen width of 768px.

[Mobile-Portfolio (1).webm](https://github.com/prince-rutagengwa/mobile-portfolio/assets/55840999/cc30abe2-17f5-4e5a-a215-bdd1acca205d)

Please update your code accordingly, making use of responsive units and testing the design across various screen sizes to ensure a consistent and responsive experience. Great job considering responsiveness and keep up the good work! 👍🌟